NineChronicles.Headless.Executable/Commands/TxCommand.cs

```diff
@@ -65,10 +70,11 @@ public TxCommand(IConsole console)
                     nameof(MonsterCollect) => new MonsterCollect(),
                     nameof(ClaimMonsterCollectionReward) => new ClaimMonsterCollectionReward(),
                     nameof(Stake) => new Stake(),
-                    // FIXME: This `ClaimStakeReward` cases need to reduce to one case.
-                    nameof(ClaimStakeReward1) => new ClaimStakeReward1(),
-                    nameof(ClaimStakeReward) => new ClaimStakeReward(),
-                    nameof(ClaimStakeReward3) => new ClaimStakeReward3(),
+                    // FIXME: How to set default action version? Can I use blockIndex instead of action version?
+                    "ClaimStakeReward" => (ActionBase)ClaimStakeRewardFactory.CreateByVersion(
+                        actionVersion ?? 2,
+                        new Address()
```

Comment:
Is it okay to pass an empty address, `new Address()`? I'm not sure why it needs an address to instantiate actions. 🤔 

(It may be overwritten by `LoadPlainValue` 🤔)

Reply:
@moreal Make sense. Let's fix this from the factory method later!

Reply:
I thought the random address will be overwritten from given plainvalue